examples/portable/executor_runner/executor_runner.cpp

```diff
@@ -29,13 +30,23 @@
 #include <executorch/runtime/executor/program.h>
 #include <executorch/runtime/platform/log.h>
 #include <executorch/runtime/platform/runtime.h>
+#ifdef ET_EVENT_TRACER_ENABLED
+#include <executorch/sdk/etdump/etdump_flatcc.h>
+#endif // ET_EVENT_TRACER_ENABLED
 
 static uint8_t method_allocator_pool[4 * 1024U * 1024U]; // 4 MB
 
 DEFINE_string(
     model_path,
     "model.pte",
     "Model serialized in flatbuffer format.");
+#ifdef ET_EVENT_TRACER_ENABLED
+DEFINE_string(
+    etdump_path,
+    "model.etdump",
+    "If ETDump generation is enabled an ETDump will be written out to this path.");
+DEFINE_uint32(num_executions, 10, "Number of times to run the model.");
```

Comment:
Do you want to make this available more generally outside EVENT_TRACER with default value = 1?